downstairs/src/region.rs

```diff
@@ -957,6 +957,7 @@ impl Extent {
     pub fn write(
         &self,
         writes: &[&crucible_protocol::Write],
+        read_fill: bool,
```

Comment:
this could be named something more specific, something like `only_write_if_unwritten`.

Reply:
Yes!
I think the "read_fill" name is here stretching it, and I want to come up with a better name not only here but for the `IOop` name as well.

Reply:
`IOop::WriteUnwritten`

And 

`write_if_unwritten`